source/modules/MEI.analytical.xml

```diff
@@ -540,6 +543,15 @@
   <classSpec ident="att.symbol.anl" module="MEI.analytical" type="atts">
     <desc xml:lang="en">Analytical domain attributes.</desc>
   </classSpec>
+  <classSpec ident="att.tabDurSym.anl" module="MEI.stringtab" type="atts">
```

Comment:
```suggestion
  <classSpec ident="att.tabDurSym.anl" module="MEI.analytical" type="atts">
```

Reply:
Schematron: Analytical domain attribute classes need to be put into the `MEI.analytical` module.